test/prepare-test.cc

```diff
@@ -625,7 +625,7 @@ template <typename T> struct user_allocator {
     return new value_type[cnt];
   }
 
-  void deallocate(pointer p, size_type cnt) { delete[] p; }
+  void deallocate(pointer p, size_type /*cnt*/) { delete[] p; }
```

Comment:
nit: I'd just remove the parameter instead of commenting out.

Reply:
Great, wasn't sure about your policy regarding this. This is my preference as well.